libs/plugin-classloader/src/main/java/org/opensearch/plugin_classloader/ExtendedPluginsClassLoader.java

```diff
@@ -30,7 +30,7 @@
  * GitHub history for details.
  */
 
-package org.opensearch.plugins;
+package org.opensearch.plugin_classloader;
```

Comment:
```suggestion
package org.opensearch.plugins.classloader;
```

Reply:
Sure but I just followed this https://github.com/opensearch-project/OpenSearch/tree/main/libs/secure-sm/src/main/java/org/opensearch `org.opensearch.secure_sm`

Reply:
Should we go with `org.opensearch.plugin_classloader;` or `org.opensearch.plugins.classloader;` ?

Reply:
I think we had `plugins` before, right?

Reply:
Ya we have already one `org.opensearch.plugins` inside `:server`.

Reply:
Looking more closely we can move the `ExtendedPluginsClassLoader` to `:server` plugins, we dont need a separate module for this.

Reply:
I have removed the `libs/plugin-classloader` and refactored to `:server`. @reta please check.

Reply:
> I have removed the `libs/plugin-classloader` and refactored to `:server`. @reta please check.

I suspect it will backfire, this module was used as security boundary to contain the `createClassloader` calls, you may need to bring it back I guess

Reply:
True it failed lot of stuff, for now I will keep the `libs/plugin-classloader`.

Reply:
@reta should we go with `org.opensearch.plugin.classloader`, looks to me like this not create split level package problem.